make/migrations/postgresql/0040_2.1.0_schema.up.sql

```diff
@@ -35,4 +35,17 @@ ALTER TABLE blob ADD COLUMN IF NOT EXISTS update_time timestamp default CURRENT_
 ALTER TABLE blob ADD COLUMN IF NOT EXISTS status varchar(255);
 ALTER TABLE blob ADD COLUMN IF NOT EXISTS version BIGINT default 0;
 CREATE INDEX IF NOT EXISTS idx_status ON blob (status);
-CREATE INDEX IF NOT EXISTS idx_version ON blob (version);
\ No newline at end of file
+CREATE INDEX IF NOT EXISTS idx_version ON blob (version);
+
+CREATE TABLE IF NOT EXISTS p2p_preheat_policy (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name varchar(255) NOT NULL,
+    description varchar(1024),
+    project varchar(255) NOT NULL,
```

Comment:
why not use project_id?

Reply:
@chlins 

@mmpei is right, let's switch to use project id in DAO